app/models/user.rb

```diff
@@ -565,6 +565,47 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def latest_activity
+    return unless registered == true
+
+    latest_activity = last_comment_at || last_article_at || latest_article_updated_at ||
+      last_reacted_at || profile_updated_at || last_moderation_notification
+
+    if latest_activity == Time.zone.today
+      "Today #{latest_activity.strftime('%b, %Y')}"
+    elsif latest_activity == Date.yesterday
+      "Yesterday #{latest_activity.strftime('%b, %Y')}"
+    else
+      latest_activity.strftime("%d %b, %Y")
+    end
+  end
+
+  def current_status
+    if suspended?
+      "Suspended"
+    elsif warned?
+      "Warned"
+    elsif trusted?
+      "Trusted"
+    else
+      "Good standing"
+    end
+  end
+
+  def current_roles
```

Comment:
In the designs for the Member Index View, we only show the below roles in the "Status" column, however another suspended role does exist -- a user can also be "Comment suspended." Should we include "Comment suspended" as well, or leave the `current_roles` as-is?

cc @anujbhavsar96 @aitchiss @Ridhwana